### PR TITLE
Add network-specific info to tokens

### PR DESCRIPTION
**Pull request by 991231**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -36,15 +36,24 @@ export interface NativeTokenInfo {
...
+  extra?: Record<string, any>;
+}
+
+export interface TokenNetworkInfo {
+  chain: SupportedChain;
+  network: string;
+  decimals: number;
+  address?: string;
+  baseUnit?: string;
+  extra?: Record<string, any>;
```
Comment (dks333) [inline src/types.ts:56]: Maybe add comments here on what kind of extra information will be in this record?

**Review by dks333 — CHANGES_REQUESTED** [review]
Good work!

File: src/types.ts (hunk 1 above)
Comment (991231) [reply src/types.ts:56]: Actually, we don't have any information to add to `extra` now. We have it here just in case, as discussed in a previous PR.

Comment (991231) [reply src/types.ts:56]: The only possible use case that I can think of right now is maybe a token has a different full name on a specific network.

Comment (dks333) [reply src/types.ts:56]: kk

**Pull request merged**